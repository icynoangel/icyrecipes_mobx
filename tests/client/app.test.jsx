import React from 'react';
import ReactDOM from 'react-dom';
import App from './../../src/client/js/app';
import {shallow} from 'enzyme';
import enzymeToJson from 'enzyme-to-json';

describe('<App />', function() {
  it('Should render <App />', () => {
    const wrapper = shallow(<App />);
    expect(enzymeToJson(wrapper)).toMatchSnapshot();
  });
});
