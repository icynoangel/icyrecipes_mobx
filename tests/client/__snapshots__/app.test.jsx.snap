// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should render <App /> 1`] = `
<div
  className="app"
>
  <inject-Buttons-with-ItemsStore />
  <inject-Notification-with-ItemsStore />
  <inject-Items-with-ItemsStore />
</div>
`;
